modules/tinymce/src/themes/silver/test/ts/headless/modes/ScrollingContextTest.ts

```diff
@@ -34,6 +34,26 @@ describe('headless.modes.ScrollingContextTest', () => {
       const div = SugarElement.fromHtml('<div style="overflow: scroll;">A</div>');
       assert.isTrue(ScrollingContext.isScroller(div), 'Should be a scroller');
     });
+
+    it('TINY-9226: overflow-x: scroll - a scroller', () => {
```

Comment:
Might be worth to add test cases for `overflow:hidden`?

Reply:
According to https://developer.mozilla.org/en-US/docs/Web/CSS/overflow, `overflow:clip` is a possible case